examples/ticks_and_spines/tick_label_right.py

```diff
@@ -0,0 +1,33 @@
+"""
+============================================
+Set default y-axis tick labels on the right
+============================================
+
+Example to show how to change two new properties in rcParams to put yaxis tick
+lables on either left or right by default.
```

Comment:
I think the text should not refer to "new" properties (unless you plan to edit this in a year).  Also some formatting changes. (Note I didn't check the comment below for length).

Otherwise looks great!

```python
"""
We can use :rc:`ytick.labelright` (default False) and :rc:`ytick.right` (default False) 
and :rc:`ytick.labelleft` (default True) and :rc:`ytick.left` (default False) to 
control where on the axes ticks and their labels appear.  These properties can also be 
set in the ``.matplotlib/matplotlibrc``.

"""
```